components/label.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Savage Component: Label
+ *
+ * @package Savage
+ */
+
+if ( ! isset( $args ) || ! is_array( $args ) ) {
+	return;
+}
+
+$defaults = [
+	'label' => '',
+];
+
+$args = wp_parse_args( $args, $defaults );
+
+// Return early if title isn't defined.
+if ( empty( $args['label'] ) ) {
+	return;
+}
+
+// Add card classname.
+savage_card_add_classname( 'savage-has-label' );
+
+printf(
+	'<p class="savage-card-label">%s</p>',
+	wp_kses( $args['label'], [] )
```

Comment:
Legge til så vi `<br />` er tillatt?

Reply:
I admin er det en enkelt tekstfelt for å legge inn label, og det skal vel helst ikke være så lang tekst der. Så jeg tenker at det ikke er nødvendig. Men du har kanskje sett noen tilfeller der det kan være nyttig?

Reply:
Har ikke sett noen tilfeller. Hvis det ikke skal være noe markup her kan vi bruke `esc_html` istedet for.